bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -420,148 +533,209 @@ Bridge netatmo:netatmoapi:home "Netatmo API" [ clientId="*********", clientSecre
+Number               Outdoor_RfStatus                  "RfStatus [%.0f / 5]"                                        <signal>           { channel = "netatmo:NAModule1:home:outside:RfStatus" }
+Switch               Outdoor_LowBattery                "LowBattery [%s]"                                            <siren>            { channel = "netatmo:NAModule1:home:outside:LowBattery" }
+Number               Outdoor_BatteryVP                 "BatteryVP [%.0f %%]"                                        <battery>          { channel = "netatmo:NAModule1:home:outside:BatteryVP" }
+DateTime             Outdoor_TimeStamp                 "TimeStamp [%1$td.%1$tm.%1$tY %1$tH:%1$tM]"                  <calendar>         { channel = "netatmo:NAModule1:home:outside:TimeStamp" }
+DateTime             Outdoor_LastMessage               "LastMessage [%1$td.%1$tm.%1$tY %1$tH:%1$tM]"                <text>             { channel = "netatmo:NAModule1:home:outside:LastMessage" }
+
+# Rain Module
+Number:Length        Rain_Hour                         "Rain Last Hour [%.03f\"]"                                   <rain>             {channel="netatmo:NAModule3:home:rain:SumRain1"}
+Number:Length        Rain_Today                        "Rain Today [%.03f\"]"                                       <rain>             {channel="netatmo:NAModule3:home:rain:SumRain24"}
+Number:Length        Rain_Week                         "Rain This Week [%.03f\"]"                                   <rain>             {channel="netatmo:NAModule3:home:rain:SumRainThisWeek"}
+Number:Length        Rain_Month                        "Rain This Month [%.03f\"]"                                  <rain>             {channel="netatmo:NAModule3:home:rain:SumRainThisMonth"}
+Number               Rain_BatteryVP                    "Rain battery status [%d%%]"                                 <battery>          {channel="netatmo:NAModule3:home:rain:BatteryVP"}
 ```
 
 
 ## sitemaps/netatmo.sitemap
 
 ```
 sitemap netatmo label="Netatmo"
-{
     Frame label="Indoor" {
-        Text    item=Netatmo_Indoor_Temperature
-        Text    item=Netatmo_Indoor_Humidity
-        Text    item=Netatmo_Indoor_Humidex  valuecolor=[<20.1="green",<29.1="blue",<28.1="yellow",<45.1="orange",<54.1="red",>54.1="maroon"]
-        Text    item=Netatmo_Indoor_HeatIndex
-        Text    item=Netatmo_Indoor_Dewpoint
-        Text    item=Netatmo_Indoor_DewpointDepression
-        Text    item=Netatmo_Indoor_Co2  valuecolor=[<800="green",<1000="orange",<1400="red",>1399="maroon"]
-        Text    item=Netatmo_Indoor_Pressure
-        Text    item=Netatmo_Indoor_AbsolutePressure
-        Text    item=Netatmo_Indoor_Noise
-        Text    item=Netatmo_Indoor_WifiStatus
-        Text    item=Netatmo_Indoor_TimeStamp
-        Text    item=Netatmo_Indoor_Location
-        Text    item=Netatmo_Indoor_LastStatusStore
+        Item=Indoor_Temp
```

Comment:
You put `Item=xxx` in all lines.
The expected syntax is `Text item=xxx`

Reply:
done